src/Map/Map.tsx

```diff
@@ -34,34 +37,20 @@ export interface PointOptions {
 
 export interface Props {
   token: string
-  sourceJson?: string
-  style?: string
-  size?: Size
-  startPosition?: StartPosition
+  sourceJson?: GeoJSON
+  initialState?: InitialState
   cluster?: boolean
   clusterOptions?: ClusterOptions
   pointOptions?: PointOptions
 }
 
-const Wrapper = styled('div')<Size>`
+const Wrapper = styled('div')<InitialState>`
```

Comment:
why not 
```js
${({ width, height} => ({ width, height }))}
```
or
```js
  height: ${({ height }) => height};
  width: ${({ width }) => width};
```
since name do match, but i guess is just being picky so feel free to skip this comment?